### PR TITLE
New keymap

### PR DESCRIPTION
**Pull request by ulounge**
Hi,

Just added my keymap file.

### CODE REVIEW

File: src/posts/keymaps/ulounge.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+keybindings: []
+keyboard: Ferris
+keyCount: 34
+keymapImage: https://raw.githubusercontent.com/ulounge/zmk-urchin/refs/heads/master/assets/images/My_Uchin.drawio.svg
+keymapUrl: https://github.com/ulounge/zmk-urchin
+languages: [French, English]
+layerCount: 7
+OS: ["Windows", "Android"]
+stagger: columnar
+summary: "See write up link for more info."
```
Comment (precondition) [inline src/posts/keymaps/ulounge.md:21]:
```suggestion
summary: "Keymap designed to balance French and English typing efficiency. Thumb-operated sticky, tap-dance  and hold-tap layers provide quick access to accents, shortcuts, callum-style mods, and media controls. See write up link for more info." 
```

Simply saying “see write up for more info” is not very enticing so may I suggest adding a short blurb?

Comment (ulounge) [reply src/posts/keymaps/ulounge.md:21]: Perfect ! Thanks for the input.

**Review by ulounge — COMMENTED** [review]
All good for me. Do you need anything else from me?

**Comment by precondition** [comment]
メルシー

Latest commit at date of merge:
https://github.com/ulounge/zmk-urchin/tree/fc3d27c6bc97800063a8bb4b77069a9861523138

**Pull request merged**